go.mod

```diff
@@ -54,4 +54,5 @@ require (
 	google.golang.org/protobuf v1.25.0
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/kothar/go-backblaze.v0 v0.0.0-20191215213626-7594ed38700f
+	istio.io/pkg v0.0.0-20201214221311-89c9b1cade38
```

Comment:
?

Reply:
Let me fix this, was trying to add a bit of logging for debugging, seems like it pulled the wrong pkg.